scripts/deploy-kit.js

```diff
@@ -13,7 +13,7 @@ const KreditsKit = artifacts.require('KreditsKit')
 
 const arapp = require('../arapp.json')
 const environment = argv['network'] || argv['environment'] || 'development'
-const apm = arapp.environments[environment].apm
+const apm = arapp.environments[environment].apm || 'open.aragonpm.eth'
```

Comment:
`arapp.environments[environment].apm` has to be an object now, so the first condition here is not going to work anymore.

Reply:
what does not work anymore? 

that `apm` entry in the arapp.json (removed in this PR) was added/used by us - and not aragon related. 
We had used that entry when `open.aragonpm.eth` was not available on all networks (and we had to use `aragonpm.eth` locally. 

now we can default to `open.aragonpm.eth`

Reply:
Please read my original PR description and follow the link. The entire reason for opening this PR was an exception about this particular `arapp.json` property having to be an object now.

Reply:
> that `apm` entry in the arapp.json (removed in this PR) was added/used by us - and not aragon related.

That would explain why it's not documented. They introduced this property themselves in the meantime, but with different keys.

Reply:
this is a WIP PR, isn't it? 
I was trying to debug that issue and get this update running here.

that configuration conflict is not solved, yet and the tests do not run, yet.

re: `apm` entry in the arapp.json file I tend to not have the apm registry configurable anymore. We now can rely on `open.aragonpm.eth` (locally and for all Ethereum netwroks we want to deploy to)

Reply:
Not sure how your comment relates to what I'm trying to explain. The config issue was solved by me removing the custom `apm` property, because Aragon now requires it to be their own documented object as a value (the one with the IPFS configs). The test failures are a different topic.

I explained these things in the original PR description, with headlines and separate paragraphs and all.

Reply:
To summarize my one comment on this one line of code again:

`arapp.environments[environment].apm` cannot be a string with an APM address anymore, as per their documentation. And it fails with an exception when trying to execute any aragon-cli command that encounters this config property as a string.

I hadn't seen that it was used in other places, so what fixed the aragon commands for me was just to remove the keys from `arapp.json`.

Reply:
which broke the setup/deployment of kredits and that's what I am trying to fix here... 
It seems you did not change any code there?!

And I know that `apm` is now a aragon related property and can not be used and I also read your comments... 

maybe I should better make a PR to your PR then we don't need to somewhat discuss WIPs and avoid missunderstandings.

Reply:
I cannot know what you know, because I don't have a telepathic link to you. Until this last comment it seemed like you hadn't understood my original comment. And my original comment was *merely a pointer* to that config option being outdated.

If you already understand something that someone points out, then why not just say "Yeah, I know, I'm working on that"? Or even nothing at all, if it's of no consequence to my involvement.

Reply:
to me it seems you did not understand what that `apm` was used for and why your change broke our deploy script. 
but I also don't know what you know, because I also don't have a telepathic link to you.

anyhow this discussion now seems to be unrelated to this PR and also on already outdated code, so I am resolving this now.